modules/calibration/data/dev_kit_advanced_ne-b/lidar_dag/lidar.dag

```diff
@@ -0,0 +1,41 @@
+# Define all coms in DAG streaming.
+module_config {
+    module_library : "/apollo/bazel-bin/modules/drivers/lidar_surestar/libsurestar_driver_component.so"
+
+    components {
+      class_name : "CompSureStarDriver"
+      config {
+        name : "surestar_driver"
+        config_file_path : "/apollo/modules/drivers/lidar/conf/surestar.conf"
+      }
+    }
+}
+
+module_config {
+    module_library : "/apollo/bazel-bin/modules/drivers/lidar_surestar/libsurestar_driver_component.so"
+
+    components {
+      class_name : "CompSurestarConvert"
+      config {
+        name : "surestar_convert"
+       
+        config_file_path : "/apollo/modules/drivers/lidar/conf/surestar.conf"
+        readers {channel: "/apollo/sensor/lidar16/Scan"}
+      }
+    }
+}
+
+module_config {
+    module_library : "/apollo/bazel-bin/modules/drivers/velodyne/compensator/libvelodyne_compensator_component.so"
+
+    components {
+      class_name : "CompensatorComponent"
+      config {
+        name : "velodyne_compensator"
+        config_file_path : "/apollo/modules/drivers/lidar/conf/velodyne16_fusion_compensator.pb.txt"
```

Comment:
please rename file name velodyne16_fusion_compensator.pb.txt to lidar16_fusion_compensator.pb.txt"